src/main/java/org/webshar/hrms/model/db/Employee.java

```diff
@@ -10,11 +10,13 @@
 import javax.persistence.Table;
 import java.util.Date;
 import java.util.Objects;
+import lombok.ToString;
 
 @Entity
 @Table(name = "employee")
 @Getter
```

Comment:
Same here lets use `@data`

Reply:
@vikramwable ,
Initially, I thought about using `@Data` in these cases but later discarded this idea mainly for entity classes due to the below limitations. 

- By defaults, it adds `HashCodeEquals` on all of the fields which are not required here. And If we want to exclude from `HashCodeEquals` then I have to annotate multiple fields to exclude them. Thus instead of that, I choose to annotate only required one `Getter` and `Setter`.
- Due to this, the developer can not rely on `@Data` and he has to make a thoughtful decision on properly implementing HashCodeEquals
- Also `@Date`  does not provide the ability to specify the parameter `callSuper`. For which one needs to again explicitly specify it by adding `@ToString(callSuper = true)`

Since my intended behavior is not getting fulfilled completely by `@Data`, I choose not to use it.
Ref: https://projectlombok.org/features/Data

Reply:
Yeah I agreed here for entity classes we should not use that but wherever we can use this lets use e.g. all request and response classes we should use them. As we hardly/never compares those class objects.

Reply:
Ok, I will handle those for request and response classes then merge this PR.

Reply:
Thank you